src/etc/inc/util.inc

```diff
@@ -1107,7 +1107,11 @@ function is_private_ip($iptocheck)
 
 function format_bytes($bytes)
 {
-    if ($bytes >= (1024 * 1024 * 1024)) {
+    if ($bytes >= (1024 * 1024 * 1024 * 1024 * 1024)) {
```

Comment:
I was thinking just for showing off, err, reducing eye fatigue...should we use "**" instead?

```suggestion
    if ($bytes >= (1024 ** 4)) {
```

see https://www.php.net/manual/en/language.operators.arithmetic.php